config/kraft/controller.properties

```diff
@@ -39,10 +39,6 @@ controller.quorum.voters=1@localhost:9093
 #     listeners = PLAINTEXT://your.host.name:9092
 listeners=PLAINTEXT://:9093
 
-# Listener name, hostname and port the broker will advertise to producers and consumers.
-# If not set, it uses the value for "listeners".
-#advertised.listeners=PLAINTEXT://your.host.name:9092
-
```

Comment:
Remove this `advertised.listeners` section in controller-only role to avoid confusion, because there must be no advertised listeners when running just a KRaft controller. ref:  [KAFKA-13456](https://issues.apache.org/jira/browse/KAFKA-13456)